api/src/main/java/org/pmiops/workbench/api/DataBrowserController.java

```diff
@@ -0,0 +1,292 @@
+
+
+        List<Concept> conceptList;
+
+        // If Concept name do search on name
+
+        if (conceptName != null) {
+            conceptList = conceptDao.findConceptLikeName(conceptName);
+        }
+        else {
+            conceptList = conceptDao.findConceptsOrderedByCount();
+        }
+
+        ConceptListResponse resp = new ConceptListResponse();
+        resp.setItems(conceptList.stream().map(TO_CLIENT_CONCEPT).collect(Collectors.toList()));
+
+        return ResponseEntity.ok(resp);
+    }
+
+    /**
+     * This method gets concepts with maps to relationship in concept relationship table
+     *
+     * @param conceptId
+     * @param conceptNum
+     * @return
+     */
+    @Override
+    public ResponseEntity<ConceptListResponse> getConceptsMapsTo(
+            Long conceptId,
+            Long conceptNum) {
+
+
+        List<Concept> conceptList;
+
+        // If Concept num is 2 , children
+        if (conceptNum == 2 ) {
+            conceptList = conceptDao.findConceptsMapsToChildren(conceptId);
+        }
+        else {
+            conceptList = conceptDao.findConceptsMapsToParents(conceptId);
+        }
+
+        ConceptListResponse resp = new ConceptListResponse();
+        resp.setItems(conceptList.stream().map(TO_CLIENT_CONCEPT).collect(Collectors.toList()));
+
+        return ResponseEntity.ok(resp);
+    }
+    /**
+     * This method searches concepts
+     *
+     * @param analysisId
+     * @param stratum1
+     * @param stratum2
+     * @param stratum3
+     * @param stratum4
+     * @param stratum5
+     * @return
+     */
+    @Override
+    public ResponseEntity<AnalysisResultListResponse> getAnalysisResults(
+            Long analysisId,
+            String stratum1,
+            String stratum2,
+            String stratum3,
+            String stratum4,
+            String stratum5) {
+
+        final List<AnalysisResult> resultList;
+
+        if ( (stratum1 != null && !stratum1.isEmpty()) && (stratum2 != null && !stratum2.isEmpty()))  {
+             resultList = analysisResultDao.findAnalysisResultsByAnalysisIdAndStratum1AndStratum2(analysisId, stratum1, stratum2);
+        }
+        else if ((stratum1 != null && !stratum1.isEmpty())) {
+            resultList = analysisResultDao.findAnalysisResultsByAnalysisIdAndStratum1(analysisId, stratum1);
+        }
+        else {
+            resultList = analysisResultDao.findAnalysisResultsByAnalysisId(analysisId);
+        }
+
+        AnalysisResultListResponse resp = new AnalysisResultListResponse();
+        resp.setItems(resultList.stream().map(TO_CLIENT_ANALYSIS_RESULT).collect(Collectors.toList()));
+
+
+        return ResponseEntity.ok(resp);
+    }
+
+    @Override
+    public ResponseEntity<org.pmiops.workbench.model.AnalysisResult> getParticipantCount() {
+        long aid =  1; // partipant count analysis id
+        final List<AnalysisResult> resultList  =  analysisResultDao.findAnalysisResultsByAnalysisId(aid);
```

Comment:
Does this need to return a list since you only use the first object? Can i just return the object?